File: cpp/677A.cpp
// 677 A. Vanya and Fence

#include <iostream>
using namespace std;

int main() {
    int n, h;
    cin >> n >> h;
    int width = 0;
    
    for (int i = 0; i < n; i++) {
        int height;
        cin >> height;
        if (height > h) width += 2;
        else width += 1;
    }
    
    cout << width;
    return 0;
}

File: java/677A.java
// 677 A. Vanya and Fence

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int h = sc.nextInt();
        int width = 0;

        for(int i=0; i<n; ++i) {
            int height = sc.nextInt();
            if(height > h) width +=2;
            else width +=1;
        }

        System.out.println(width);
        sc.close();
    }
}
